University of Wisconsin-Madison Libraries Catalog.js

```diff
@@ -135,9 +149,9 @@ var testCases = [
 					}
 				],
 				"date": "2011",
-				"abstractNote": "xi, 159 pages : illustrations ; 23 cm",
 				"callNumber": "PN171.F56 P83 2011",
 				"libraryCatalog": "University of Wisconsin-Madison Libraries Catalog",
+				"numPages": "159",
 				"publisher": "Chicago : Association of College and Research Libraries, 2011.",
```

Comment:
sorry, just noticed this -- we can do better with this, too, right? At least the publisher/place division I'd expect to be consistent and the year we can at least chop off when it follows this pattern (if this becomes too much of a hack, may be worth rethinking and going to the MARC view instead).